serde-generate/tests/test_utils.rs

```diff
@@ -51,7 +51,7 @@ pub enum SerdeData {
     UnitVector(Vec<()>),
     SimpleList(SimpleList),
     CStyleEnum(CStyleEnum),
-    ComplexMap(BTreeMap<([u32; 2], [u8; 4]), ()>),
+    ComplexMap(BTreeMap<(), Vec<(u8, u8, u8)>>),
```

Comment:
I had to change this test not because of serialization/deserialization, but because the `equals()` comparison function in Dart from the [`test`](https://pub.dev/packages/test) package doesn't work on tuples and map keys.

If you say the test should remain the same, I can write a custom matcher class from that package, though it would introduce hundreds of lines of more code just for the comparison purpose.

Reply:
Thanks for the explanation. Changing the test is not ideal because the feature won't be tested as much in other languages (existing or future).